docs/manual/java/gettingstarted/GettingStartedSbt.md

```diff
@@ -21,7 +21,7 @@ To create your project, follow these steps:
 
 1. Enter the following command to invoke `sbt new` using the Lagom Giter8 template:
    ```
-   sbt -Dsbt.version=0.13.16 new https://github.com/lagom/lagom-java.g8
+   sbt new lagom/lagom-java.g8
```

Comment:
Since we're now recommending sbt 1.x this command can be simplified (no need to explicitly set the version or the transport).

Reply:
I'm OK with simplifying this, but I'm not 100% sure that we won't continue to run into users having trouble with this.

Whether this command works depends on the version of the _sbt launcher_ you have installed, and our experience has shown that many people (even Lightbend engineers) don't know which version of the launcher they are using because of sbt's ability to automatically switch versions depending on the `sbt.version` of the project you are working on.

Anyway, let's try this and then watch what happens.

Some other options we can try if users still report trouble:

1. More strongly encourage use of the [Tech Hub starter templates](https://developer.lightbend.com/start/?group=lagom) which include a bundled sbt launcher.
2. Include a troubleshooting section that specifically references the errors you might see with the wrong launcher version (and suggests updating the sbt installation with a pointer to sbt docs)